src/App.jsx

```diff
@@ -1,88 +1,16 @@
-import React, { useState, useEffect } from 'react';
+import React, { useState } from 'react';
 import axios from 'axios';
-import track from './utils/analytics';
-import store from './utils/jwt-store';
-import feature from './utils/flaggie';
-import styles from './App.module.scss';
-import Search from './components/Search';
+import { Route } from 'react-router';
 import Nav from './components/Nav';
-import PopularDestinations from './components/PopularDestinations';
-import Attractions from './components/Attractions';
-import Hero from './components/Hero/Hero';
-import Modal from './components/Modal/Modal';
-
-axios.interceptors.request.use((config) => {
```

Comment:
This was moved to Home.jsx, let's instead abstract it out into its' own config and then execute it in either App.jsx or index